pkg/features/featureflags.go

```diff
@@ -0,0 +1,180 @@
+package features
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+// Flags feature flags
+var Flags *Features
+var getVersion func() string
+
+func init() {
+	Flags = &Features{}
+	getVersion = getLinuxPTPPackageVersion
+}
+
+// Features ...
+type Features struct {
+	oc          bool
+	bc          bool
+	logSeverity bool
+	gm          bool
+	gmHoldover  bool
+	syncE       bool
+	dualBC      bool
+	twoPortOC   bool
+	ha          bool
+	leapFile    bool
+	bcHoldover  bool
+}
+
+// Versions ...
+const (
+	LinuxPTPVersion3112 = "3.1.1-2.el8_6.3"
```

Comment:
there is a possible this will bump minor version

Reply:
That's why I added the comparison lib stuff, as we would not have to track each version. 

However we do have to maintain the comparison lib so that's the trade off.